src/main/java/com/kodilla/stream/forumuser/ForumUser.java

```diff
@@ -0,0 +1,50 @@
+package com.kodilla.stream.forumuser;
+
+import java.time.LocalDate;
+
+public final class ForumUser {
+    private final int userId;
+    private final String userName;
+    private  char sex = 'M'/ 'F';
```

Comment:
a co to jest? Przede wszystkich private final. 
Po drugie, typ char jest konwertowalny na int, a wiec 'M' / 'F' to jest dzielenie dwoch liczb. Tu ma byc jeden znak, kry ma byc ustawiony w konstruktorze.